gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/ViewWithUiHandlers.java

```diff
@@ -44,7 +44,7 @@
      */
     protected C getUiHandlers() {
         if (uiHandlers == null) {
-            logger.severe("uiHandlers are not set.  Did you call getUiHandlers() from your view's constructor?");
+            logger.severe("uiHandlers are not set. You should call setUiHandlers() from your presenter's constructor.");
```

Comment:
You could add that setUiHandlers must be called first from the ctor in order to have an instance available.